test/UnitTest/v1/test_casesNGSIv1.py

```diff
@@ -16,704 +16,974 @@
         resp=json.loads(resInJson)
-        #print(resp)
+        print(resp)
         resp=resp['subscribeResponse']
         sid=resp['subscriptionId']
-        #print(sid)
+        print(sid)
 
-	#update the created entity
+        #update the created entity
         url=brokerIp+"/ngsi10/updateContext"
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata23),headers=headers)
-        #print(r.status_code)
         resp_content1=r.content
         resInJson=resp_content1.decode('utf8').replace("'",'"')
         resp1=json.loads(resInJson)
-        #print(resp1)
+        print(resp1)
 
-	#validate via accumulator
-        url="http://0.0.0.0:8888/validateNotification"
+        #validate via accumulator
+        url="http://180.179.214.211:8888/validateNotification"
         r=requests.post(url,json={"subscriptionId" : sid})
         print(r.content)
         assert r.status_code == 200
-        
+
 #testCase 13
 '''
-  Tseting subscription for its if and else part : 2) for User - Agent Header
+  To test subscription for its if and else part : 2) for User - Agent Header
 '''
 def test_getSubscription18():
-	#create an entity
+        #create an entity
         url=brokerIp+"/ngsi10/updateContext"
-	headers={'Content-Type' : 'application/json'}
+        headers={'Content-Type' : 'application/json'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata24),headers=headers)
-        #print(r.status_code)
         resp_content1=r.content
         resInJson=resp_content1.decode('utf8').replace("'",'"')
         resp1=json.loads(resInJson)
-	#print(resp1)
+        print(resp1)
 
-	#subscribing
-	url=brokerIp+"/ngsi10/subscribeContext"
+        #subscribing
+        url=brokerIp+"/ngsi10/subscribeContext"
         headers= {'Content-Type': 'application/json','User-Agent' : 'lightweight-iot-broker'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata25),headers=headers)
-        #print(r.status_code)
         resp_content=r.content
         resInJson= resp_content.decode('utf8').replace("'", '"')
         resp=json.loads(resInJson)
-        #print(resp)
+        print(resp)
         resp=resp['subscribeResponse']
         sid=resp['subscriptionId']
-        #print(sid)
+        print(sid)
 
-	#update created entity
+        #update created entity
         url=brokerIp+"/ngsi10/updateContext"
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata26),headers=headers)
-        #print(r.status_code)
         resp_content1=r.content
         resInJson=resp_content1.decode('utf8').replace("'",'"')
         resp1=json.loads(resInJson)
-        #print(resp1)
+        print(resp1)
 
-	#validate via accumulator
-        url="http://0.0.0.0:8888/validateNotification"
+        #validate via accumulator
+        url="http://180.179.214.211:8888/validateNotification"
         r=requests.post(url,json={"subscriptionId" : sid})
         print(r.content)
         assert r.status_code == 200
-        
+
 #testCase 14
 '''
```

Comment:
Confirm the notification payload in the absence and presence of Require-Reliability.